const-oid/src/encoder.rs

```diff
@@ -132,7 +132,7 @@ const fn base128_byte(arc: Arc, pos: usize, total: usize) -> Result<u8> {
     debug_assert!(pos < total);
     let last_byte = checked_add!(pos, 1) == total;
     let mask = if last_byte { 0 } else { 0b10000000 };
-    let shift = checked_sub!(checked_sub!(total, pos), 1) * 7;
+    let shift = checked_mul!(checked_sub!(checked_sub!(total, pos), 1), 7);
```

Comment:
This fixes this warning:
```
warning: arithmetic operation that can potentially result in unexpected side-effects
   --> const-oid/src/encoder.rs:135:17
    |
135 |     let shift = checked_sub!(checked_sub!(total, pos), 1) * 7;
    |                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#arithmetic_side_effects
note: the lint level is defined here
   --> const-oid/src/lib.rs:11:5
    |
11  |     clippy::arithmetic_side_effects,
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

I don't think the shift should overflow in the first place, but clippy still warns about it.